docker-compose.yaml

```diff
@@ -28,6 +28,7 @@ services:
       APPINSIGHTS_INSTRUMENTATIONKEY: ${APPINSIGHTS_INSTRUMENTATIONKEY:-notset}
       REDIS_HOSTNAME: redis
       REDIS_PORT: 6379
+      STORAGE_FACILITIES_PERCENTAGE: 50
```

Comment:
Does adding the variable to this docker-compose file enable it across all environments? Or will it have to be set in config files in the deployment pipeline? If it's the latter we should start the next adding-value runbook as part of this change to include it.

Reply:
Yes it will mean we need to make changes across all environments. We probably should start the next adding-value runbook.